lib/workers/pr/changelog/release-notes.ts

```diff
@@ -3,42 +3,71 @@ import { linkify } from 'linkify-markdown';
 import MarkdownIt from 'markdown-it';
 
 import { api } from '../../../platform/github/gh-got-wrapper';
+import { api as api_gitlab } from '../../../platform/gitlab/gl-got-wrapper';
 import { logger } from '../../../logger';
 import { ChangeLogResult, ChangeLogNotes } from './common';
 
 const { get: ghGot } = api;
+const { get: glGot } = api_gitlab;
 
 const markdown = new MarkdownIt('zero');
 markdown.enable(['heading', 'lheading']);
 
 export async function getReleaseList(
-  githubApiBaseURL: string,
+  apiBaseURL: string,
   repository: string
 ): Promise<ChangeLogNotes[]> {
   logger.trace('getReleaseList()');
   // istanbul ignore if
-  if (!githubApiBaseURL) {
+  if (!apiBaseURL) {
+    logger.debug('No apiBaseURL');
     return [];
   }
   try {
-    let url = githubApiBaseURL.replace(/\/?$/, '/');
-    url += `repos/${repository}/releases?per_page=100`;
-    const res = await ghGot<
-      {
-        html_url: string;
-        id: number;
-        tag_name: string;
-        name: string;
-        body: string;
-      }[]
-    >(url);
-    return res.body.map(release => ({
-      url: release.html_url,
-      id: release.id,
-      tag: release.tag_name,
-      name: release.name,
-      body: release.body,
-    }));
+    let url = apiBaseURL.replace(/\/?$/, '/');
+    if (apiBaseURL.search(/github/) !== -1) {
+      // github repo
+      url += `repos/${repository}/releases?per_page=100`;
+      const res = await ghGot<
+        {
+          html_url: string;
+          id: number;
+          tag_name: string;
+          name: string;
+          body: string;
+        }[]
+      >(url);
+      return res.body.map(release => ({
+        url: release.html_url,
+        id: release.id,
+        tag: release.tag_name,
+        name: release.name,
+        body: release.body,
+      }));
+    }
+    if (apiBaseURL.search(/gitlab/) !== -1) {
+      // not github, hopefully gitlab
+      url += `projects/${repository.replace(
+        /\//g,
+        '%2f'
+      )}/releases?per_page=100`;
+      const res = await glGot<
+        {
+          name: string;
+          release: string;
+          description: string;
+          tag_name: string;
+        }[]
+      >(url);
+      return res.body.map(release => ({
+        url: undefined, // FIXME: url should be set
```

Comment:
Can we point to the release page here?
```suggestion
        url: `${baseUrl}/${repository}/-/releases/${release.tag_name}`
```

`baseUrl` needs to be passed

Reply:
`baseUrl` is not defined in that scope, used `apiBaseURL` and the releases path instead, hope that's what you meant.